exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/limit/TestLimitWithExchanges.java

```diff
@@ -125,6 +125,15 @@ public void testLimitImpactExchange() throws Exception {
     }
   }
 
+  @Test
+  public void TestLimitAllOnParquet() throws Exception {
```

Comment:
update the tested query. I did not notice the query in unit test is slightly different than the one I used in sqlline.